html/e2etest.js

```diff
@@ -0,0 +1,654 @@
+	} else if(substream === 1) {
+		toastr.success("Switched simulcast substream! (normal quality)", null, {timeOut: 2000});
+		$('#sl-2').removeClass('btn-primary btn-success').addClass('btn-primary');
+		$('#sl-1').removeClass('btn-primary btn-info btn-success').addClass('btn-success');
+		$('#sl-0').removeClass('btn-primary btn-success').addClass('btn-primary');
+	} else if(substream === 2) {
+		toastr.success("Switched simulcast substream! (higher quality)", null, {timeOut: 2000});
+		$('#sl-2').removeClass('btn-primary btn-info btn-success').addClass('btn-success');
+		$('#sl-1').removeClass('btn-primary btn-success').addClass('btn-primary');
+		$('#sl-0').removeClass('btn-primary btn-success').addClass('btn-primary');
+	}
+	// Check the temporal layer
+	if(temporal === 0) {
+		toastr.success("Capped simulcast temporal layer! (lowest FPS)", null, {timeOut: 2000});
+		$('#tl-2').removeClass('btn-primary btn-success').addClass('btn-primary');
+		$('#tl-1').removeClass('btn-primary btn-success').addClass('btn-primary');
+		$('#tl-0').removeClass('btn-primary btn-info btn-success').addClass('btn-success');
+	} else if(temporal === 1) {
+		toastr.success("Capped simulcast temporal layer! (medium FPS)", null, {timeOut: 2000});
+		$('#tl-2').removeClass('btn-primary btn-success').addClass('btn-primary');
+		$('#tl-1').removeClass('btn-primary btn-info btn-success').addClass('btn-success');
+		$('#tl-0').removeClass('btn-primary btn-success').addClass('btn-primary');
+	} else if(temporal === 2) {
+		toastr.success("Capped simulcast temporal layer! (highest FPS)", null, {timeOut: 2000});
+		$('#tl-2').removeClass('btn-primary btn-info btn-success').addClass('btn-success');
+		$('#tl-1').removeClass('btn-primary btn-success').addClass('btn-primary');
+		$('#tl-0').removeClass('btn-primary btn-success').addClass('btn-primary');
+	}
+}
+
+// Custom functions to use with Insertable streams: the senderTransform
+// function is what we use to "encrypt" the media we send, while the
+// receiverTransform is what we use to "decrypt" the media we receive.
+// Both functions will use a secret that we'll prompt for when opening
+// the page, meaning that, ideally, we'll be the only one able to decrypt
+// it: Janus will not have access to the media. In a scenario involving
+// multiple participants (e.g., a video call or conference), to use these
+// same functions you'll want to distribute the secret among all participants
+// using an out of band mechanism, so that they can encrypt and decrypt
+// the media and keep Janus out of the loop. Notice that these functions
+// are just an example, and are basically exact copies from this demo:
+// 		https://github.com/webrtc/samples/blob/gh-pages/src/content/peerconnection/endtoend-encryption/js/main.js
+// That said, you're free (or actually, encouraged) to try and use other
+// custom, and more advanced, trasform functions as well (e.g., SFrames).
```

Comment:
typo: trasform

Reply:
No `n` in Italian (trasformare), so I'll play the "not my first language" card :laughing: